src/clients/memberclient.cpp

```diff
@@ -187,6 +192,14 @@ void submit_open_network(RpcTlsClient& tls_connection)
   cout << response.dump() << std::endl;
 }
 
+void submit_rekey_ledger(RpcTlsClient& tls_connection)
```

Comment:
I think it would have been ok for only the HTTP tests to do this via scurl, since we're going to drop FTCP/the memberclient post 0.7.

Reply:
True. But since our python infra supports both seamlessly, it was cheap enough to add the code just here.

Reply:
Ok!